common_tools/verify.py

```diff
@@ -0,0 +1,122 @@
+import os
+import tempfile
+import numpy as np
+from typing import List
+from multiprocessing import Pool, cpu_count, Manager
+import argparse
+from utils import *
+
+def compile_attention_mlir_file(mlir_file : str, target_device: str, hip_target: str, extra_flags: list):
+    vmfb_file = os.path.splitext(mlir_file)[0] + ".vmfb"
+    exec_args = [
+    "iree-compile",
+    # Input file
+    f"{mlir_file}",
+    # Output file
+    "-o",
+    f"{vmfb_file}",
+    # Target Device: hip
+    f"--iree-hal-target-device={target_device}",
+    # Device: MI300x
+    f"--iree-hip-target={hip_target}",
+    ] + extra_flags
+    ret_value, stdout, stderr = run_iree_command(exec_args)
+    if ret_value == 0:
+        print(f"Successfully compiled {mlir_file} to {vmfb_file}")
+    else:
+        print(f"Failed to compile {mlir_file}")
+        return None
+    return vmfb_file
+
+def get_output_npy(vmfb_file : str, inputs : List[str], target_device: str):
+    with tempfile.NamedTemporaryFile(suffix='.npy', delete=False) as out:
```

Comment:
We usually use bin files for validation. (We've actually noticed a difference between the two in some special cases 😄). Want to change to bin?